Gemfile

```
@@ -37,7 +37,7 @@ gem 'pager_api'
 gem 'paper_trail'
 gem 'pg', '~> 1.0.0'
 gem 'prometheus_exporter'
-gem 'puma', '~> 3.12.6'
+gem 'puma'
```

maybe specify the versioning too? 
```suggestion
gem 'puma', '~> 5.0'
```